packages/showcase/src/app/components/tooltip/tooltip.component.html

```diff
@@ -1,12 +1,90 @@
-<runtime-bundles [bundles]="['utils/typography']" />
+<runtime-bundles
+  [bundles]="['components/tooltip', 'utils/form', 'components/checkbox']"
+/>
 
 <header>
-  <h1 class="heading-1">Tooltip</h1>
+  <h1 class="heading-1">{{ metadata.title }}</h1>
 </header>
 
-<p>
-  This page is under construction and will be available soon. You can see the
-  <a class="hyperlink" href="/old/tooltip.html" target="_blank"
-    >old version here</a
-  >.
+<p class="body-regular-l">
+  {{ metadata.description }}
 </p>
+
```

Comment:
We should include something about this:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/tooltip_role#accessibility_concerns